lib/sonar/cli/cli.rb

```diff
@@ -27,7 +29,12 @@ def usage
 
     desc 'search_all [QUERY TERM]', 'Search all Sonar record types'
     def search_all(term)
-      Sonar::Search::QUERY_TYPES_MAP.keys.each do |type|
+      if term =~ IS_IP
+        types = ip_search_type_names
+      else
+        types = domain_search_type_names
+      end
+      types.each do |type|
         search(type, term) rescue 'Search failed'
```

Comment:
Probably won't be too difficult.  Do you think it should hide all errors or just check for "Data not found" and hide those?